restdocs-api-spec-gradle-plugin/build.gradle.kts

```diff
@@ -8,6 +8,7 @@ plugins {
     kotlin("jvm")
     `java-gradle-plugin`
     `kotlin-dsl`
+    id("com.gradle.plugin-publish") version "0.10.1"
```

Comment:
Needs to be version 0.12.0 now or this error will occur:

```
> Task :restdocs-api-spec-gradle-plugin:publishPlugins FAILED
Could not reuse POM from pluginMaven publication because mavenCoordinates() was used
Publishing plugin com.epages.restdocs-api-spec version 0.9.9-SNAPSHOT

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':restdocs-api-spec-gradle-plugin:publishPlugins'.
> Failed to post to server.
  Server responded with:
  This version of the com.gradle.plugin-publish plugin is no longer supported due to a critical security vulnerability.
  Please update to the latest version of the com.gradle.plugin-publish plugin found here:
     https://plugins.gradle.org/plugin/com.gradle.plugin-publish
  You can read more about this vulnerability here:
     https://blog.gradle.org/plugin-portal-update
```

Reply:
Ack, changed.